src/test/regress/expected/pg14.out

```diff
@@ -226,5 +226,43 @@ SELECT attname || ' ' || attcompression FROM pg_attribute WHERE attrelid::regcla
  {"a p","a p","b ","b "}
 (2 rows)
 
+-- ALTER STATISTICS .. OWNER TO CURRENT_ROLE
+CREATE TABLE st1 (a int, b int);
+CREATE STATISTICS role_s1 ON a, b FROM st1;
+SELECT create_distributed_table('st1','a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+ALTER STATISTICS role_s1 OWNER TO CURRENT_ROLE;
+CREATE ROLE role_1 WITH LOGIN SUPERUSER;
```

Comment:
would this test work on enterprise? We may need to adjust the output to work fine on both